src/pages/docs/kubernetes/targets/kubernetes-agent/upgrading.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-08-22
+modDate: 2024-08-22
+title: Upgrading the agent
+navTitle: Overview
+navSection: Kubernetes agent
+description: How a Kubernetes Agent is upgraded
+navOrder: 10
+---
+
+Recent changes to the Kubernetes-Agent Helm Chart have necessitated a breaking change, meaning existing installations
+of the v1 Kubernetes-Agent helm chart are (currently) unable to automatically upgrade to thew new version (v2).
+
+Installed v1 instances will continue to operate as expected, however no further development work (bugfixes or enhancements)
+will be performed on the v1 helm chart. While you may continue to use v1 of the helm-chart, it is highly recommended to perform a manual upgrade to v2
```

Comment:
Did you mean to say 'won't be performed on the v1'?

Reply:
Is the message here different from what we discussed earlier? We create a sense of urgency here to update.

Reply:
I've updated a minor tweak to imply v1 gets no further updates - what more do you want to say to generate impetus to upgrade?

Reply:
I'll add comments to the new version.